battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/gamecontroller/NormalController.java

```diff
@@ -70,8 +69,8 @@ public void startGame() {
     }
 
     @Override
-    public TurnStatus turnStatus(Player player) {
-        return new TurnStatus(getBoardForPlayer(player), isMyTurn(player), getWinner(), 1);
+    public Response turnStatus(Player player) {
+        return Response.turnStatus(getBoardForPlayer(player), isMyTurn(player), getWinner(), 1);
```

Comment:
Magic :1234:  It would be nice to extract `1` to constant with some meaningful name. :ghost: